.yarn/versions/e0af1fb6.yml

```diff
@@ -0,0 +1,10 @@
+releases:
+  "@radix-ui/react-roving-focus": patch
+  "@radix-ui/react-toggle-group": patch
+
+declined:
+  - primitives
+  - "@radix-ui/react-menu"
+  - "@radix-ui/react-radio-group"
+  - "@radix-ui/react-tabs"
+  - "@radix-ui/react-toolbar"
```

Comment:
Declined these because functionality wise they're identical to before.

Reply:
I'm always curious about this, I know the result is the same, but don't they need to be published still so you don't end up with duplicate dependencies potentially?

Reply:
> but don't they need to be published still so you don't end up with duplicate dependencies potentially?

What's your thinking here? Sounds like something I've not considered.

Reply:
I'm just thinking that if some packages aren't updated, then they will be pointing to the previous version of the roving focus package, as opposed to the newly published one, which will be pointing to the newer one. So it means someone would end up with multiple versions of it their dependency tree, so less code sharing, etc.

Reply:
ah interesting... no harm in `patch`ing them I guess. I'll do that 👍